conf/th/thailand01.yml

```diff
@@ -481,30 +481,38 @@ years:
   - public_holiday: true
     date: '2022-04-14'
     names:
-      en: วันสงกรานต์
-      th: วันสงกรานต์
+      en: songkran
+      th: songkran
   - public_holiday: true
     date: '2022-04-15'
     names:
-      en: วันสงกรานต์
-      th: วันสงกรานต์
+      en: songkran
+      th: songkran
+   - public_holiday: true
+    date: '2022-05-02'
+    names:
+      en: Labour Day in lieu
+      th: Labour Day in lieu
+   - public_holiday: true
+    date: '2022-05-04'
+    names:
+      en: h.m. King's Coronation
+      th: h.m. King's Coronation
   - public_holiday: true
     date: '2022-07-28'
     names:
-      en: วันเฉลิมพระชนมพรรษาสมเด็จพระเจ้าอยู่หัวมหาวชิราลงกรณ บดินทรเทพยวรางกูร
-      th: วันเฉลิมพระชนมพรรษาสมเด็จพระเจ้าอยู่หัวมหาวชิราลงกรณ บดินทรเทพยวรางกูร
```

Comment:
I understand changing the `en:` version to English, but why are we removing the `th:` one to English too?

Reply:
The TH names were added in by Alex's gem, but were incorrect, and a company of user flagged they were not in any commonly spoken Thai language, and in fact English is more useful. Also previously we've never included TH names in this file so we're not losing anything previously offered

Reply:
Haha, wow, what an interesting fact, I wonder who wrote them! Thanks Joe